### PR TITLE
Feature integrate all delete apis

### PR DESCRIPTION
**Pull request by purisaurabh**
This feature branch contains : 
          1) All delete apis integration

### CODE REVIEW

File: src/Constants.js [hunk 1]
```diff
@@ -49,11 +49,24 @@ export const UPDATE_EXPERIENCE_ENDPOINT =
 export const UPDATE_CERTIFICATE_ENDPOINT =
   "/api/profiles/:profile_id/certificates/:certificate_id";
 
+export const DELETE_PROFILE_ENDPOINT = "/api/profiles/:profile_id";
+export const DELETE_ACHIEVEMENT_ENDPOINT =
+  "/api/profiles/:profile_id/achievement/:achievement_id";
```
Comment (Payal13) [inline src/Constants.js]: wrong API path - 
achievement it should be plural, achievements, projects, experiences etc

File: src/api/experienceApi.jsx [hunk 2]
```diff
@@ -1,10 +1,11 @@
 import { createApi } from "@reduxjs/toolkit/query/react";
 import {
   CREATE_EXPERIENCE_ENDPOINTS,
+  DELETE_EXPERIENCE_ENDPOINT,
```
Comment (Payal13) [inline src/api/experienceApi.jsx:4]: always try to add new additions in the last

File: src/components/Builder/Achievement/index.jsx [hunk 3]
```diff
@@ -216,8 +244,8 @@ const Achievement = ({ achievementData }) => {
                     rules={[
                       {
                         required: true,
-                        message: "Name is required",
-                      },
+                        message: "Name is required"
```
Comment (Payal13) [inline src/components/Builder/Achievement/index.jsx]: I think you guys are using different eslink or some settings or commas in the last getting added and sometimes getting removed. Please check together why this is happening.

File: src/components/Builder/Achievement/index.jsx [hunk 4]
```diff
@@ -270,8 +298,26 @@ const Achievement = ({ achievementData }) => {
           />
         </SortableContext>
       </DndContext>
+      <Modal
```
Comment (Payal13) [inline src/components/Builder/Achievement/index.jsx]: move this confirmation modal in common components and use

File: src/components/Builder/Certification/index.jsx [hunk 5]
```diff
@@ -350,12 +378,26 @@ const Certification = ({ certificationData }) => {
           />
         </SortableContext>
       </DndContext>
+      <Modal
```
Comment (Payal13) [inline src/components/Builder/Certification/index.jsx]: use from common component

File: src/Constants.js (hunk 1 above)
Comment (Abhishek-Jain-1925) [reply src/Constants.js]: Done!

File: src/api/experienceApi.jsx (hunk 2 above)
Comment (Abhishek-Jain-1925) [reply src/api/experienceApi.jsx:4]: During Formatting the document, it follows Alphabetical order for imports
That's why Delete done last but import came first order

File: src/components/Builder/Achievement/index.jsx (hunk 3 above)
Comment (Abhishek-Jain-1925) [reply src/components/Builder/Achievement/index.jsx]: Done!

File: src/components/Builder/Achievement/index.jsx (hunk 4 above)
Comment (Abhishek-Jain-1925) [reply src/components/Builder/Achievement/index.jsx]: Done!

File: src/components/Builder/Certification/index.jsx (hunk 5 above)
Comment (Abhishek-Jain-1925) [reply src/components/Builder/Certification/index.jsx]: Done!

**Review by Payal13 — APPROVED** [review]

**Pull request merged**